examples/sagemaker_inference_agent/requirements.in

```diff
@@ -0,0 +1,4 @@
+git+https://github.com/flyteorg/flytekit.git@fb8f04a59dd8c709c35d9adc6889b7ed227c2aaa#subdirectory=plugins/flytekit-aws-sagemaker
```

Comment:
after the flytekit PR is merged, need to modify the package name before merging this PR.